CMakeLists.txt

```diff
@@ -1,56 +1,34 @@
 cmake_minimum_required(VERSION 3.19)
-project(xania)
-
-####
-# Shell out to the Makefile to get all the paths it installed for us
-set(CMAKE_MAKE_CONFIG_FILE ${CMAKE_CURRENT_BINARY_DIR}/config-from-make.cmake)
-execute_process(
-        COMMAND make cmake-print-deps CMAKE_CONFIG_FILE=${CMAKE_MAKE_CONFIG_FILE}
-        WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
-        RESULT_VARIABLE MAKE_CONFIG_FAIL)
-if (MAKE_CONFIG_FAIL)
-    message(FATAL_ERROR "Unable to get makefile configuration")
-endif ()
-include(${CMAKE_MAKE_CONFIG_FILE})
-####
 
 set(BUILD_VERSION "(local build)" CACHE STRING "Build version (build number only)")
 set(BUILD_FULL_VERSION "(local build)" CACHE STRING "Full string of build version (with git info)")
 set(BUILD_COMMIT_URL "https://this.is.a.local.build/1234" CACHE STRING "Full string of git URL to this build")
 
+project(xania)
+
 set(CMAKE_CXX_STANDARD 20)
-add_compile_options(-Werror -Wall -Wextra)
+add_compile_options(-Werror -Wall -Wextra -Wno-error=deprecated-declarations)
```

Comment:
We're not using `[[deprecated]]` in any of the mud sources,  are you getting errors related to this from a library or something?

Reply:
yes I've had it with fmt upto date. That said, I will probably remove this from the PR in it final form as it have nothing to do in  it (compile flags can be overriden at configuration)

Reply:
ok

Reply:
FYI the warning is issued by lyra (using the current master)
xania/src/doorman/main.cpp:33:70: warning: 'std::string lyra::detail::result_base::errorMessage() const' is deprecated [-Wdeprecated-declarations]
   33 |         fmt::print("Error in command line: {}\n", result.errorMessage());
      |                                                   ~~~~~~~~~~~~~~~~~~~^~
In file included from /tmp/lyra_install/include/lyra/parser_result.hp
...
                 from /tmp/lyra_install/include/lyra/lyra.hpp:12,
                 from /home/marie/trucaseb/git/xania/src/doorman/main.cpp:21:
/tmp/lyra_install/include/lyra/detail/result.hpp:22:36: note: declared here
   22 |         [[deprecated]] std::string errorMessage() const { return message(); }
Not sure anything is needed on xania's side, I give it for the record...